lib/config/index.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
we should test for valid json. still type checking, but good practice to demonstrate.